### DISCUSSION TITLE
Github Copilot alternative

### DISCUSSION THREAD

**Discussion by NeoRusi** [Cyfrin/foundry-full-course-cu#834] [Q&A]
Hey guys was looking the section from foundry stable coin part and seemed the amazing features of Github Copilot 
can anyone suggest a good working free AI pair programming assistant ?

**Comment by 0xusmanf** [comment]
I am using [Codeium](https://marketplace.visualstudio.com/items?itemName=Codeium.codeium), It supports 70+ programming languages and is also free. So far I don't have any complaints about it.
